docs/testing.md

```diff
@@ -29,7 +30,7 @@ Then, from the `extensions/ql-vscode` directory, use the appropriate command to
 * View Tests: `npm test:view`
 * VSCode integration tests: `npm run test:vscode-integration`
 
-#### CLI integration tests
+#### Running CLI integration tests from the terminal
```

Comment:
I've renamed these headings because of markdownlint's rule about no headings with duplicate names.